modules/analysis-common/src/test/resources/rest-api-spec/test/analysis-common/30_tokenizers.yml

```diff
@@ -133,7 +101,7 @@
             text: "foobar"
             explain: true
             tokenizer:
-              type: nGram
+              type: ngram
```

Comment:
same here, this is about the tokenizer, but we shouldn't use the camel case name here regardless